CONFIGURATION.md

```diff
@@ -11,6 +11,7 @@ Your configuration overrides the default configuration, which can be found
 gitlabApi: https://gitlab.example.com/api/v4
 
 # Required
+# Can be generated here: https://gitlab.com/profile/personal_access_tokens At least api_read scope is needed
```

Comment:
Please use `https://gitlab.example.com/profile/personal_access_tokens` to make clear that you cannot get access tokens for your own instance on gitlab.com